testing/scenario_app/ios/Scenarios/Scenarios.xcodeproj/project.pbxproj

```diff
@@ -45,6 +45,7 @@
 		68D4017D2564859300ECD91A /* ContinuousTexture.m in Sources */ = {isa = PBXBuildFile; fileRef = 68D4017C2564859300ECD91A /* ContinuousTexture.m */; };
 		68D5003F291ED645001ACFE1 /* golden_platform_view_cliprrect_with_transform_iPhone 8_16.0_simulator.png in Resources */ = {isa = PBXBuildFile; fileRef = 68D5003D291ED645001ACFE1 /* golden_platform_view_cliprrect_with_transform_iPhone 8_16.0_simulator.png */; };
 		68D50042291ED8CD001ACFE1 /* golden_platform_view_cliprect_with_transform_iPhone 8_16.0_simulator.png in Resources */ = {isa = PBXBuildFile; fileRef = 68D50041291ED8CD001ACFE1 /* golden_platform_view_cliprect_with_transform_iPhone 8_16.0_simulator.png */; };
+		68D93AEE2A46097E0054AB6D /* golden_platform_view_with_negative_backdrop_filter_iPhone SE (3rd generation)_16.2_simulator.png in Resources */ = {isa = PBXBuildFile; fileRef = 68D93AED2A46097E0054AB6D /* golden_platform_view_with_negative_backdrop_filter_iPhone SE (3rd generation)_16.2_simulator.png */; };
```

Comment:
Can you link to an example run of the scenario app test being run on HEAD? Regenerating these goldens for the branch SGTM

Reply:
Sure: 

https://ci.chromium.org/ui/p/flutter/builders/prod/Mac%20Production%20Engine%20Drone/104625/overview, search "29. test: Scenario App Integration Tests"

Reply:
A task called linux_mac_unopt should be responsible to spawn multiple build for iOS tests. I couldn't find linux_mac_unopt in this PR.

Reply:
@drewroengoogle do you know why only release_build="true" targets are running in presubmit on release PRs? I would expect all the targets to be run. In this instance, we use the tests to regenerate the goldens.

Reply:
I filed https://github.com/flutter/flutter/issues/131279 for tracking. @cyanglaz is there a way to regenerate these locally, or do we need to use LUCI for the goldens?

Reply:
These needs to be regenerated locally from the contributor. 

Should we wait until the tests are enabled before re-generating the goldens so that we can ensure the tests pass?

Reply:
Hey, can you point me to some resources that can guide me on how to regenerate the tests locally? Thanks :)

Reply:
https://github.com/flutter/engine/blob/main/testing/scenario_app/README.md

For this particular case, you would need to set the simulator to iPhone 8 with iOS version as 16.0 while running the tests.
You will then get some test failures, where the new images are attached. After that, you can download the image, rename it with the correct pattern and copy it to the test folder. 

Let me know if you have issues with it. If it is too much trouble for you, I can do it for you when I get a chance :)

Reply:
>Would it make the most sense to cherry pick the current ci.yaml in main into here?

Should we have a separate cherry-pick for updating the ci.yaml?

Reply:
@CaseyHillers Is there any objection to me updating ci.yaml to the most recent version? The reason I ask is because I'm not familiar with issues that might occur from doing this, like if there are any new targets in main that shouldn't actually be running for this candidate branch.

Reply:
Yes. Please avoid blanket copy/pastes of it. Only update the minimum amount that is necessary.

It's ok to CP ci.yaml changes due to infra reasons, otherwise, we shouldn't. Flutter's CI needs to be able to verify what we're shipping to users is working based on what has already been shipped, not based on what will ship.

Reply:
@cyanglaz I am unable to run this command locally: 
`ninja -C out/ios_debug_sim_unopt`

I am getting 
```
ninja: Entering directory `out/ios_debug_sim_unopt'
[42/2573] OBJCXX obj/third_party/skia/src/gpu/ganesh/mtl/gpu.GrMtlGpu.o
FAILED: obj/third_party/skia/src/gpu/ganesh/mtl/gpu.GrMtlGpu.o 
../../buildtools/mac-x64/clang/bin/clang++ -MD -MF obj/third_party/skia/src/gpu/ganesh/mtl/gpu.GrMtlGpu.o.d -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D_FORTIFY_SOURCE=2 -D_LIBCPP_DISABLE_AVAILABILITY=1 -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCPP_ENABLE_THREAD_SAFETY_ANNOTATIONS -D_DEBUG -DSK_ENABLE_DUMP_GPU -DSK_DISABLE_AAA -DSK_LEGACY_IGNORE_DRAW_VERTICES_BLEND_WITH_NO_SHADER -DSK_DISABLE_LEGACY_SHADERCONTEXT -DSK_DISABLE_LOWP_RASTER_PIPELINE -DSK_FORCE_RASTER_PIPELINE_BLITTER -DSK_METAL_WAIT_UNTIL_SCHEDULED -DSK_DISABLE_EFFECT_DESERIALIZATION -DSK_ENABLE_SKSL -DSK_ENABLE_PRECOMPILE -DSK_ASSUME_GL_ES=1 -DSK_GANESH -DSK_LEGACY_LAYER_BOUNDS_EXPANSION -DSK_ENABLE_API_AVAILABLE -DSK_GAMMA_APPLY_TO_A8 -DSKIA_IMPLEMENTATION=1 -DSK_METAL -I../.. -Igen -I../../third_party/libcxx/include -I../../third_party/libcxxabi/include -I../../build/secondary/third_party/libcxx/config -I../../third_party/skia -isysroot /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.0.sdk -mios-simulator-version-min=11.0 -fno-strict-aliasing -arch x86_64 -fcolor-diagnostics -Wall -Wextra -Wendif-labels -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-but-set-parameter -Wno-unused-but-set-variable -Wno-implicit-int-float-conversion -Wno-c99-designator -Wno-deprecated-copy -Wno-psabi -Wno-unqualified-std-cast-call -Wno-non-c-typedef-for-linkage -Wno-range-loop-construct -Wunguarded-availability -Wno-deprecated-declarations -fvisibility=hidden -Wstring-conversion -Wnewline-eof -O0 -g2 -fvisibility-inlines-hidden -std=c++17 -fno-rtti -nostdinc++ -nostdinc++ -fvisibility=hidden -fno-exceptions -stdlib=libc++ -fobjc-arc -fvisibility-inlines-hidden -fobjc-call-cxx-cdtors -std=c++17 -fno-rtti -nostdinc++ -nostdinc++ -fvisibility=hidden -fno-exceptions  -c ../../third_party/skia/src/gpu/ganesh/mtl/GrMtlGpu.mm -o obj/third_party/skia/src/gpu/ganesh/mtl/gpu.GrMtlGpu.o
../../third_party/skia/src/gpu/ganesh/mtl/GrMtlGpu.mm:1594:27: error: incompatible types casting 'dispatch_semaphore_t' (aka 'dispatch_semaphore_s *') to 'const void *' with a __bridge_retained cast
    const void* cfFence = (__bridge_retained const void*) semaphore;
                          ^                        ~~~~~  ~~~~~~~~~
../../third_party/skia/src/gpu/ganesh/mtl/GrMtlGpu.mm:1600:38: error: incompatible types casting 'const void *' to 'dispatch_semaphore_t' (aka 'dispatch_semaphore_s *') with a __bridge cast
    dispatch_semaphore_t semaphore = (__bridge dispatch_semaphore_t)cfFence;
                                     ^         ~~~~~~~~~~~~~~~~~~~~ ~~~~~~~
2 errors generated.
[55/2573] CXX obj/third_party/skia/src/shaders/skia.SkPictureShader.o
ninja: build stopped: subcommand failed.

```

I am using xcode 15 beta. Any pointers?

Reply:
> I am using xcode 15 beta. Any pointers?

Could you try to use a stable XCode version? (14.3.1?). There are several XCode 15 beta issues that we are still resolving.

Reply:
> @cyanglaz I am unable to run this command locally: `ninja -C out/ios_debug_sim_unopt`
> 
> I am getting
> 
> ```
> ninja: Entering directory `out/ios_debug_sim_unopt'
> [42/2573] OBJCXX obj/third_party/skia/src/gpu/ganesh/mtl/gpu.GrMtlGpu.o
> FAILED: obj/third_party/skia/src/gpu/ganesh/mtl/gpu.GrMtlGpu.o 
> ../../buildtools/mac-x64/clang/bin/clang++ -MD -MF obj/third_party/skia/src/gpu/ganesh/mtl/gpu.GrMtlGpu.o.d -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D_FORTIFY_SOURCE=2 -D_LIBCPP_DISABLE_AVAILABILITY=1 -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCPP_ENABLE_THREAD_SAFETY_ANNOTATIONS -D_DEBUG -DSK_ENABLE_DUMP_GPU -DSK_DISABLE_AAA -DSK_LEGACY_IGNORE_DRAW_VERTICES_BLEND_WITH_NO_SHADER -DSK_DISABLE_LEGACY_SHADERCONTEXT -DSK_DISABLE_LOWP_RASTER_PIPELINE -DSK_FORCE_RASTER_PIPELINE_BLITTER -DSK_METAL_WAIT_UNTIL_SCHEDULED -DSK_DISABLE_EFFECT_DESERIALIZATION -DSK_ENABLE_SKSL -DSK_ENABLE_PRECOMPILE -DSK_ASSUME_GL_ES=1 -DSK_GANESH -DSK_LEGACY_LAYER_BOUNDS_EXPANSION -DSK_ENABLE_API_AVAILABLE -DSK_GAMMA_APPLY_TO_A8 -DSKIA_IMPLEMENTATION=1 -DSK_METAL -I../.. -Igen -I../../third_party/libcxx/include -I../../third_party/libcxxabi/include -I../../build/secondary/third_party/libcxx/config -I../../third_party/skia -isysroot /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.0.sdk -mios-simulator-version-min=11.0 -fno-strict-aliasing -arch x86_64 -fcolor-diagnostics -Wall -Wextra -Wendif-labels -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-but-set-parameter -Wno-unused-but-set-variable -Wno-implicit-int-float-conversion -Wno-c99-designator -Wno-deprecated-copy -Wno-psabi -Wno-unqualified-std-cast-call -Wno-non-c-typedef-for-linkage -Wno-range-loop-construct -Wunguarded-availability -Wno-deprecated-declarations -fvisibility=hidden -Wstring-conversion -Wnewline-eof -O0 -g2 -fvisibility-inlines-hidden -std=c++17 -fno-rtti -nostdinc++ -nostdinc++ -fvisibility=hidden -fno-exceptions -stdlib=libc++ -fobjc-arc -fvisibility-inlines-hidden -fobjc-call-cxx-cdtors -std=c++17 -fno-rtti -nostdinc++ -nostdinc++ -fvisibility=hidden -fno-exceptions  -c ../../third_party/skia/src/gpu/ganesh/mtl/GrMtlGpu.mm -o obj/third_party/skia/src/gpu/ganesh/mtl/gpu.GrMtlGpu.o
> ../../third_party/skia/src/gpu/ganesh/mtl/GrMtlGpu.mm:1594:27: error: incompatible types casting 'dispatch_semaphore_t' (aka 'dispatch_semaphore_s *') to 'const void *' with a __bridge_retained cast
>     const void* cfFence = (__bridge_retained const void*) semaphore;
>                           ^                        ~~~~~  ~~~~~~~~~
> ../../third_party/skia/src/gpu/ganesh/mtl/GrMtlGpu.mm:1600:38: error: incompatible types casting 'const void *' to 'dispatch_semaphore_t' (aka 'dispatch_semaphore_s *') with a __bridge cast
>     dispatch_semaphore_t semaphore = (__bridge dispatch_semaphore_t)cfFence;
>                                      ^         ~~~~~~~~~~~~~~~~~~~~ ~~~~~~~
> 2 errors generated.
> [55/2573] CXX obj/third_party/skia/src/shaders/skia.SkPictureShader.o
> ninja: build stopped: subcommand failed.
> ```
> 
> I am using xcode 15 beta. Any pointers?

I got the same error, did you solve it?

Reply:
> > I am using xcode 15 beta. Any pointers?
> 
> Could you try to use a stable XCode version? (14.3.1?). There are several XCode 15 beta issues that we are still resolving.

I got the same error and the XCode version i use is 14.3.1